typescript/formsApps.d.ts

```diff
@@ -501,6 +501,7 @@ export type FormsAppConfiguration<
   isGoogleLoginSupported: boolean
   isClientLoggingEnabled: boolean
   recaptchaPublicKey: string
+  googleMapsKey?: string
```

Comment:
I know recaptcha is named like so, but can we update this to more accurately represent what is stored i.e.

```suggestion
  googleMapsIntegrationKeyId?: string
```

Reply:
Ah yeah derp